kornia/geometry/depth.py

```diff
@@ -365,3 +366,36 @@ def depth_warp(
     warper = DepthWarper(pinhole_dst, height, width, align_corners=align_corners)
     warper.compute_projection_matrix(pinhole_src)
     return warper(depth_src, patch_dst)
+
+
+def depth_from_disparity(disparity: Tensor, baseline: Union[float, Tensor], focal: Union[float, Tensor]) -> Tensor:
+    """Computes depth from disparity.
+
+    Args:
+        disparity: Disparity tensor of shape :math:`(B, 1, H, W)`.
+        baseline: float/tensor containing the distance between the two lenses.
+        focal: float/tensor containing the focal length.
+
+    Return:
+        Depth map of the shape :math:`(B, 1, H, W)`.
```

Comment:
Why the empty dim here? why not just (B,H,W)?

Reply:
The stereo.py file uses the size of (B, 1, H, W). Also, since it could be a `disparity to/from depth` it might be better to keep the dimensions of the 2 same.

Reply:
Probably accepting (*,H,W) would make everyone happy. @pri1311 we can adjust the others if needed too

Reply:
Alright!

Also, I just realized, if we use focal and baseline of size batch_size, it won't broadcast currently given the current implementation, i.e. it won't be doing batch multiplication. Need to tweak the code to do BMM.